rollup-common.js

```diff
@@ -64,12 +80,28 @@ export function litRollupConfig({ entryPoints, external = [] } = options) {
   // of all our code in a single file, tell Terser to minify that, and then throw
   // it away. This seeds the name cache in a way that guarantees every property
   // gets a unique mangled name.
-  const nameCache = {};
+  const nameCache = {
+    props: {
+      // Note all properties in the terser name cache are prefixed with '$'
+      // (presumably to avoid collisions with built-ins).
```

Comment:
Maps are useful :)

Reply:
When you're using the CLI this is written to a JSON file, so I guess that's why they do it this way.